### PR TITLE
avm fritz binding for merge

### PR DESCRIPTION
**Pull request by robbyb67**
The other PR was closed as I did a hard reset on my repo. Nevertheless should this commit be ok. :-)

### CODE REVIEW

**Comment by kaikreuzer** [comment]
Thanks. Doesn't look too bad at a first glance, but you have missed the Signed-off footer on your commit... Could you amend your commit please?

**Comment by kaikreuzer** [comment]
Hm, a few more comments, while browsing quickly over the code:
You forgot to mention the included libraries (jackson etc.) in the about.html. See the "Third-Party Content About File Template" at https://eclipse.org/legal/epl/about.php on how to add that.

File: addons/binding/org.openhab.binding.avmfritz/ESH-INF/thing/bridge.xml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+				<label>IP address</label>
+				<description>The local IP address or hostname of the FRITZ!Box
+				</description>
+				<default>fritz.box</default>
+			</parameter>
+			<parameter name="port" type="integer" required="false" min="0"
+				max="65335">
+				<label>Port</label>
+				<description>Port of the FRITZ!Box</description>
+				<default></default>
```
Comment (kaikreuzer) [inline addons/binding/org.openhab.binding.avmfritz/ESH-INF/thing/bridge.xml]: this must be removed!

File: addons/binding/org.openhab.binding.avmfritz/ESH-INF/thing/thing-types.xml [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+			<channel id="outlet" typeId="outlet" />
+		</channels>
+
+		<config-description>
+			<parameter name="ipAddress" type="text" required="true">
+				<context>network-address</context>
+				<label>IP address</label>
+				<description>The local IP address or hostname of the FRITZ!Box
+				</description>
+				<default>fritz.box</default>
```
Comment (kaikreuzer) [inline addons/binding/org.openhab.binding.avmfritz/ESH-INF/thing/thing-types.xml]: No, this actually registers as fritz.powerline.

File: addons/binding/org.openhab.binding.avmfritz/ESH-INF/thing/thing-types.xml [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+			</parameter>
+			<parameter name="port" type="integer" required="false" min="0"
+				max="65335">
+				<label>Port</label>
+				<description>Port of the FRITZ!Box</description>
+				<default></default>
```
Comment (kaikreuzer) [inline addons/binding/org.openhab.binding.avmfritz/ESH-INF/thing/thing-types.xml]: This line must be removed!

**Comment by robbyb67** [comment]
Any other comments or ready to sqash / signoff ?

**Comment by kaikreuzer** [comment]
If you have covered my comment reg. about.html, I'd say ready to squash!

**Comment by robbyb67** [comment]
Good hit on about.html - thx a lot ;-)

**Comment by kaikreuzer** [comment]
Thanks, merged!
Feel free to open new PRs for any further fixes and changes!

**Pull request merged**